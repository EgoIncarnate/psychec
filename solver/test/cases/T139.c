void x() {
    f();
    T1 at = f;
    T2 bt = *f;
    T3 ct = **f;
    T4 dt = ***f;
}

void y() {
    Q1 aq = g;
    Q2 bq = *g;
    Q3 cq = **g;
    Q4 dq = ***g;
    g();
}

void z() {
    S1 as = h;
    S2 bs = *h;
    h();
    S3 cs = **h;
    S4 ds = ***h;
}
