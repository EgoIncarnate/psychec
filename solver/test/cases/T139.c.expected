typedef  uintptr_t (* T4 ) () ;
typedef  uintptr_t (* T3 ) () ;
typedef  uintptr_t (* T2 ) () ;
typedef  uintptr_t (* T1 ) () ;
typedef  uintptr_t (* S4 ) () ;
typedef  uintptr_t (* S3 ) () ;
typedef  uintptr_t (* S2 ) () ;
typedef  uintptr_t (* S1 ) () ;
typedef  uintptr_t (* Q4 ) () ;
typedef  uintptr_t (* Q3 ) () ;
typedef  uintptr_t (* Q2 ) () ;
typedef  uintptr_t (* Q1 ) () ;
uintptr_t f () ;
uintptr_t g () ;
uintptr_t h () ;
